rdt-open-reader/src/main/java/io/ona/rdt/util/Utils.java

```diff
@@ -267,9 +268,9 @@ protected Void doInBackground(Void... voids) {
             boolean isUserAuthorized = syncUtils.verifyAuthorization();
 
             Activity currentActivity = RDTApplication.getInstance().getCurrentActivity();
-            boolean isFormActivity = currentActivity instanceof RDTJsonFormActivity;
+            boolean isFormSourceActivity = (currentActivity instanceof PatientProfileActivity) || (currentActivity instanceof PatientRegisterActivity);
```

Comment:
Why are we checking for patientprofileactivity?

Reply:
https://github.com/onaio/rdt-standard/issues/958#issuecomment-831758094